docs/usage/configuration.md

```diff
@@ -99,12 +99,18 @@ To enlarge the PVC's size follow the following steps:
 
 > Drawback of this approach: The cache's size in the Shoot spec (`providerConfig.caches[].size`) diverges from the PVC's size.
 
-#### [Alternative 2] Remove and readd the cache
+#### [Alternative 2] Remove and re-add the cache
 
-There is always the option to remove the cache from the Shoot spec and to readd it again with the updated size.
+There is always the option to remove the cache from the Shoot spec and to re-add it again with the updated size.
 
 > Drawback of this approach: The already cached images get lost and the cache starts with an empty disk.
 
+## Caching images from private repositories
```

Comment:
I assume the registry cache always sends the credentials when doing upstream requests? If that's the case, let's document it.
Can this behaviour cause requests being rejected due to invalid credentials for public images? I know this is a server-side behaviour of the upstream registry. But if there are well-known registries that work in this way, let's document them.